tests/Kros.KORM.UnitTests/Integration/DbSetShould.cs

```diff
@@ -297,6 +298,69 @@ public async Task DeleteDataAsync()
             }
         }
 
+        [Fact]
+        public async Task DeleteDataByIdAsync()
+        {
+            using (var korm = CreateDatabase(CreateTable_TestTable, InsertDataScript))
+            {
+                var dbSet = korm.Query<Person>().AsDbSet();
+                dbSet.Delete(1);
+
+                await dbSet.CommitChangesAsync();
+
+                korm.Query<Person>()
+                    .Should()
+                    .NotContain(p => p.Id == 1);
+            }
+        }
+
+        [Fact]
+        public async Task DeleteDataByIdsAsync()
+        {
+            using (var korm = CreateDatabase(CreateTable_TestTable, InsertDataScript))
+            {
+                var dbSet = korm.Query<Person>().AsDbSet();
+                dbSet.Delete(1);
+                dbSet.Delete(2);
+
+                await dbSet.CommitChangesAsync();
+
+                korm.Query<Person>().Should().BeEmpty();
+            }
+        }
+
+        [Fact]
+        public async Task DeleteDataByLinqConditionAsync()
```

Comment:
Can we add test to delete by non-primary key column (`p.ParentId == 42`)? Maybe condition with more than one column (`p.Value > 10 && p.Value < 20`)?